examples/config.yaml

```diff
@@ -74,10 +73,14 @@ listen: http://127.0.0.1:5555
 web_path_prefix: /
 
 # Optional kubectl version which provides a download link to the the binary
-kubectl_version: v1.11.2
+kubectl_version: v1.23.6
```

Comment:
Do you think it worth a try of kubectl:1.23.7 ? I see 1.23.6 has 4 critical CVEs (but maybe those are still there in 1.23.7 too.) 😅